spec/obs_github_deployments/deployment_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+RSpec.describe ObsGithubDeployments::Deployment, :vcr do
+  subject { ObsGithubDeployments::Deployment.new(repository: gh_test_repository, access_token: gh_test_access_token) }
+
+  describe "locked?" do
+    it "returns true when deployment state is 'queued'" do
+      expect(subject.locked?).to eq(true)
+    end
+
+    it "returns false when deployment is in a state different then 'queued'" do
```

Comment:
Maybe is better if we avoid mentioning implementation details here too.

Reply:
done